src/Moryx/Serialization/EntryConvert/EntryToModel/EntryCollectionBase.cs

```diff
@@ -52,7 +52,7 @@ public string CurrentType
         /// <summary>
         /// Possible entries that can be used to create instances
         /// </summary>
-        public string[] PossibleTypes => CollectionRoot.Value.Possible;
+        public string[] PossibleTypes => CollectionRoot.Value.Possible?.Select(p => p?.Key).ToArray();
```

Comment:
Can you also make this an object array instead of string, classes could also use `DisplayName` and `Description 🤔`